.github/workflows/test.yml

```diff
@@ -27,12 +31,47 @@ jobs:
           cache: "yarn"
           cache-dependency-path: "**/yarn.lock"
```

Comment:
I may not understand how what you are doing is different, but it does appear to be meaningful:
![image](https://user-images.githubusercontent.com/9324390/207945549-66158675-d513-42eb-9bde-1b942db2112b.png)

That's from the [second run](https://github.com/Ed-Fi-Exchange-OSS/Meadowlark/actions/runs/3706257180/attempts/2). Also, look at the difference in time between that and [the first run](https://github.com/Ed-Fi-Exchange-OSS/Meadowlark/actions/runs/3706257180/attempts/1), before the cache was populated: saved about a minute.

I see the `yarn install` step being skipped there, and it works out just fine. 

So .... I don't understand why this isn't working the way the `setup-node` action describes, but I'm very happy with the end result!

Reply:
I agree, it seems like setup-node should be able to do this. I'm reading [advanced-usage](https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#caching-packages-data) to see if this can be done differently. I will try this first to see if there's a difference

Reply:
After reading about setup-node and the advanced-usage, it's still not clear for me the improvement of using the global cache given the big difference in time and performance.

Currently, this is caching until the yarn-lock file changes, if desired we can add an id related to the workflow run to avoid using cache from other builds but I don't see any risks with doing it as we are doing it right now

Reply:
I don't have any concern about what is being done: it works, and the performance is better. I wonder if the node-setup version isn't handling the monorepos well.